### ISSUE TITLE
Black screen on mobile android, cant run even basic 3d sence

### ISSUE THREAD

**Issue by BachDovn** [godotengine/godot#90512]
### Tested versions

Godot 4.2.1 and I dont see solution on other version as well

### System information

window 10, godot4.2.1 vulakan mobile

### Issue description

Black screen forever when open 3d sence with mobile android. Hope it work normal

Android info:
Oppo CPH2239
Android 11, api 30
Chipset: mt6765
Build:CPH229_11_C_32
Realease: 11
OpenGL:3.0.2
CPU:arm64-v8a

VersionTemplate 4.2.1.stable

Export setting:

name="Android"
platform="Android"
runnable=true
dedicated_server=false
custom_features=""
export_filter="all_resources"
include_filter=""
exclude_filter=""
export_path="../../Godot Export/Test.apk"
encryption_include_filters=""
encryption_exclude_filters=""
encrypt_pck=false
encrypt_directory=false

[preset.0.options]

custom_template/debug=""
custom_template/release=""
gradle_build/use_gradle_build=true
gradle_build/export_format=0
gradle_build/min_sdk="24"
gradle_build/target_sdk="30"
architectures/armeabi-v7a=false
architectures/arm64-v8a=true
architectures/x86=false
architectures/x86_64=false
version/code=1
version/name=""
package/unique_name="com.example.$genname"
package/name=""
package/signed=true
package/app_category=2
package/retain_data_on_uninstall=false
package/exclude_from_recents=false
package/show_in_android_tv=false
package/show_in_app_library=true
package/show_as_launcher_app=false
launcher_icons/main_192x192=""
launcher_icons/adaptive_foreground_432x432=""
launcher_icons/adaptive_background_432x432=""
graphics/opengl_debug=true
xr_features/xr_mode=0
screen/immersive_mode=true
screen/support_small=true
screen/support_normal=true
screen/support_large=true
screen/support_xlarge=true
user_data_backup/allow=true
command_line/extra_args=""
apk_expansion/enable=false
apk_expansion/SALT=""
apk_expansion/public_key=""
permissions/custom_permissions=PackedStringArray()
permissions/access_checkin_properties=false
permissions/access_coarse_location=false
permissions/access_fine_location=false
permissions/access_location_extra_commands=false
permissions/access_mock_location=false

### Steps to reproduce

1, create project have only one 3d sence and camera and sky
2, setting export
3, run Remote debug
4, Mobile open app, show godot logo then black screen forever(I mean still run app but never show 3d sence).
![image](https://github.com/godotengine/godot/assets/132882686/8f82b129-cf1d-4589-add0-ead339bf404d)

### Minimal reproduction project (MRP)

no

**Comment by BachDovn** [comment]
@huwpascoe No. I run on window already, camera can see green static.

**Comment by AThousandShips** [comment]
Try aiming the camera down, you're probably given a different view so it might miss it on a smaller screen

**Comment by BachDovn** [comment]
@AThousandShips I dont think so sir. It happen what ever I make camera position in, all black look like never load 3d sence, My other project have loading screen, it load full, and done, change sence I can play and hear my player action but just never show what ever 3d, all black

**Comment by AThousandShips** [comment]
Not a "sir"

**Comment by AThousandShips** [comment]
What's the need to laugh? Behave respectfully or I'll refer this to the code of conduct team

**Comment by BachDovn** [comment]
@AThousandShips are you serious? I don't mean to disrespect you, maybe you just thought that yourself. OK, what ever you are.
I try render full way but still the same.

**Comment by akien-mga** [comment]
@BachDown If you wrongly assumed someone's gender (something I would simply advise never to do), and get corrected, the respectful response is "Apologies, this wasn't intentional" and not reacting with a laughing emoji.

**Comment by adamscott** [comment]
> OK, what ever you are.

@BachDovn This is not a way to answer. This is even worse. Not every contributor is by default a man, please read the [code of conduct](https://godotengine.org/code-of-conduct/).

**Comment by adamscott** [comment]
@BachDovn And please, remove that laughing react here: https://github.com/godotengine/godot/issues/90512#issuecomment-2049391666

If you don't, it will be reported to the code of conduct team.

**Comment by BachDovn** [comment]
@AThousandShips so you're not a man, you're female or lgbt. Ok, I'm sorry for not knowing your gender . But I think this gender misunderstanding is understandable

**Comment by BachDovn** [comment]
@AThousandShips So, what is your gender?

**Comment by AThousandShips** [comment]
My pronouns are on my profile clear as day, there's no honest intent here, or this would have been resolved with a reasonable apology

**Comment by BachDovn** [comment]
@AThousandShips Ok, I just call you is you....is that ok.
You right, I am sorry to you...and I hope every one here is nice. And help me solve android proplem.Thanks

**Comment by AThousandShips** [comment]
Apology accepted

**Comment by Swarkin** [comment]
interesting convo. anyways, ive been experiencing something similar.

the high end vulkan renderer works perfectly fine.
unsure about mobile but as it uses vulkan too it should be alright too.
but when using compatibility, it glitches out a lot and produces this:
## Flash warning
https://github.com/godotengine/godot/assets/102416174/e2d42a3c-97cd-4560-b934-99f4f014f63f

as compat is the default, godot is unusable on my phone. using remote deploy for my game and forcing vulkan, it works as expected.

using a Samsung S20 FE.
